authors/settings.py

```diff
@@ -121,6 +121,15 @@
     },
 ]
 
+#email settings
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = os.getenv('EMAIL_HOST')
+EMAIL_HOST_USER = os.getenv('EMAIL_HOST_USER')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
```

Comment:
Please check the environment variable for `EMAIL_HOST_PASSWORD`. It is throwing errors
<img width="1161" alt="Screenshot 2019-06-11 at 08 56 39" src="https://user-images.githubusercontent.com/38750268/59247067-df07ff80-8c26-11e9-9a50-88f63d6844f7.png">